synapse_auto_compressor/pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
```

Comment:
I guess since this is essentially a separate package we need a different pyproject for it?

Reply:
I don't remember why I did this exactly, but I think that when I tried to upgrade maturin/pyo3, it was starting to become an issue that they were not properly two different projects?